### ISSUE TITLE
Duplicate artifacts in sys_modules.yaml and kernel_modules.yaml

### ISSUE THREAD

**Issue by mnrkbys** [tclahr/uac#261]
sys_modules.yaml has the following artifatct:

```
  -
    description: List loaded kernel modules.
    supported_os: [linux]
    collector: command
    command: ls -l -a /sys/module
    output_file: ls_-l_-a_sys_module.txt
```

However, kernel_modules.yaml has the same:

```
  -
    description: Display the loaded kernel modules (compare with /proc/modules).
    supported_os: [linux]
    collector: command
    command: ls -la /sys/module
    output_file: ls_-la_sys_module.txt
```

It might be better to unify them into one.

**Comment by tclahr** [comment]
Confirmed.
I will keep kernel_modules.yaml and remove sys_modules.yaml.

**Comment by tclahr** [comment]
Merged in PR #263
